### DISCUSSION TITLE
请问可以在netlify部署waline吗？

### DISCUSSION THREAD

**Discussion by System2c** [walinejs/waline#1737] [Q&A] [answered]
如题

**Comment by musnows** [comment]
同问，vercel的访问速度有些慢，文章打开了，要加载很久才能加载出评论

**Comment by Diralpo** [comment]
同问，貌似现在vercel在国内访问不了

**Reply by musnows** [reply]
> 同问，貌似现在vercel在国内访问不了

不是访问不了，是必须要绑定自己的域名。vercel给你提供的域名被墙了，绑定自己的域名就还算快

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/guide/deploy/netlify.html
